src/renderer/contexts/SwitchDatabaseProvider.tsx

```diff
@@ -15,6 +15,14 @@ export default function SwitchDatabaseProvider({
         if (statement.analyze.type === 'use') {
           setCurrentDatabase(statement.analyze.db);
         }
+        if (
+          statement.analyze.type === 'create' ||
+          // only has delete, while drop is fired instead
+          // @ts-expect-error TODO: fix this
+          statement.analyze.type === 'drop'
```

Comment:
There is nothing we can do at the moment to avoid the typescript error besides asking typescript to ignore. The only two solutions:
- Make the PR to the node-sql-parser to update the type definition
- We provide our old definition that include the drop

Either of those two solutions requires some significant work. I believe we can ignore temporary.

You should also need to reload the schema is there is ALTER.